admin/tabs/logging/LogLevelPanel.js

```diff
@@ -47,7 +47,7 @@ export class LogLevelPanel extends Component {
         ],
         editors: [
             {field: 'name'},
-            {field: 'level'}
+            {field: 'level', type: 'lookupSelect'}
```

Comment:
Waiting on decision regarding log level lookups null handling to decide exactly how to handle this control.

If we decide to leave well enough alone and not remove 'None' as a lookup we don't have to do this here. We can just set this field to lookupStrict and get the SelectField automatically. If we remove this somewhat confusingly named lookup, we should NOT set lookupStrict(as we would be adding a null option). In which case, we need this flag to get a selectField rather than a comboField.